toolboxes/coefficientformpdes/main.cpp

```diff
@@ -2,17 +2,15 @@
  */
 
 #include <feel/feelmodels/coefficientformpdes/coefficientformpdes.hpp>
+#include <feel/feelmodels/coefficientformpdes/coefficientformpdes_registered_type.hpp>
 
 template <int nDim,int nOrderGeo>
 int
 runApplicationCoefficientFormPDEs()
 {
     using namespace Feel;
 
-    typedef FeelModels::CoefficientFormPDEs< Simplex<nDim,nOrderGeo>,
-                                             Lagrange<1,Scalar,Continuous,PointSetFekete>,
-                                             Lagrange<2,Scalar,Continuous,PointSetFekete>,
-                                             Lagrange<1,Vectorial,Continuous,PointSetFekete> > model_type;
+    using model_type = FeelModels::coefficient_form_PDEs_t< Simplex<nDim,nOrderGeo> >;
```

Comment:
would it be possible to add structured mesh (MeshStructured) @vincentchabannes . We will need that for a project coupling feelpp with NN. Unfortunately we can only use structured mesh for the moment

Reply:
it's possible, we need to compile more lib. But consequently, the global compilation time will still increase.